<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udg::QExternalApplicationConfigurationScreenBase</class>
 <widget class="QWidget" name="udg::QExternalApplicationConfigurationScreenBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>502</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>External applications</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../main/main.qrc">
    <normaloff>:/images/grid.png</normaloff>:/images/grid.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QPushButton" name="btnDelete">
     <property name="text">
      <string>Suprimeix</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnDown">
     <property name="text">
      <string>Baixa</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string>Afegeix</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="5">
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Url</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnUp">
     <property name="text">
      <string>Puja</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Les aplicacions externes obren un navegador web amb una URL personalitzada a partir de les dades de l'estudi que tingueu obert.&lt;/p&gt;&lt;p&gt;Si a la URL que proporcioneu aquí hi figura algun dels següents camps, aquests seran substituits per el valor corresponent:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;&lt;b&gt;%studyId%&lt;/b&gt; Identificador de l'estudi&lt;/li&gt;
&lt;li&gt;&lt;b&gt;%patientId%&lt;/b&gt; Identificador del pacient&lt;/li&gt;
&lt;/ul&gt;
&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../main/main.qrc"/>
  <include location="../main/main.qrc"/>
  <include location="../main/main.qrc"/>
 </resources>
 <connections/>
</ui>
